src/enums/roleTypes.ts

```diff
@@ -1,6 +1,8 @@
 export enum RoleTypes {
   COMPLETING_PARTY = 'Completing Party',
-  INCORPORATOR = 'Incorporator',
   DIRECTOR = 'Director',
-  SUBSCRIBER = 'Subscriber',
+  INCORPORATOR = 'Incorporator',
+  PROPRIETOR = 'proprietor',
+  PARTNER = 'partner',
```

Comment:
Why these 2 are not Pascal Case?

Reply:
Good question, currently the API is returning (`/parties`) these roles all lowercase, so this was in concert with that.

Reply:
![Screen Shot 2022-04-05 at 12 10 13 PM](https://user-images.githubusercontent.com/53542131/161831389-5e98917d-cbb9-4fc0-a710-3006ea4a03dc.png)

Reply:
This could impact other roles as well. eg: `incorporator`, `completing_party`. So either we need to fix the parties api or ui to consider other roles as well.

Reply:
It could be that it's different end points. The `/parties` returns lowercase, the `/directors` roles are structured a little different and they are pascal case.

Reply:
If the api is updated, we can update here as well. Currently the UI just handles both variations.

Reply:
I'm sorry, i was incorrect, the first variation is being returned via CORRECTION filing, thats where the structure differs
NOT the `/directors` endpoint

Reply:
oh! ok

Reply:
That being said, i'm not sure if the upper/lowercase will even matter. Will update the UI and see if it has an impact

Reply:
Yea, it actually doesn't matter. The UI is handling it either way. 
Yes the api is a tiny bit inconsistent between Corrections and our Change/Alteration filings, but it could also be how the value is saved from an Incorporation to begin with 🤷 
Either way, super low impact. Updated locally here for consistency, good catch!

Reply:
And for what it's worth, Create UI save the reg filing with roleType in sentence case:

![image](https://user-images.githubusercontent.com/10002889/161837759-b4ca0e04-a9e5-4603-b20c-6d223fd5512c.png)

@vysakh-menon-aot Let me know if I need to change this.

Reply:
@severinbeauvais This impacts api validation if you do so. So we need to take it as separate issue and analyze the impacts.

Reply:
The roleType above was not changed for registrations -- only extended.

The current API code seems to work: I filed the above registration and the filing output looks OK.

Reply:
Right. It won't break registration validation but report breaks. Also I was looking into IA validation and report templates. which will break

Reply:
Thanks for investigating. Is it a quick fix or do we need to coordinate multiple, dependent changes?

What of this? https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json#L74

Reply:
My goal is to identify (and keep track of) anything that needs changing/fixing.

Reply:
For sure, also following here as i don't know what the solution is. 
Should we discuss on a call tomorrow?

Reply:
Yes @severinbeauvais if we are changing it we need to start from schema to fixing existing filing_json data in Prod. Since the schema is defined to use Pascal Case we can update `/parties and `/directors` endpoint to provide Pascal Case role.

Reply:
How about we chat at 10am tomorrow (after splanning)?

Reply:
sure

Reply:
What we decided:
- API should return role types according to schema (title case); Vysakh will create a ticket for this and will ask Linda whether it's OK to do it this sprint
- Filings UI will need its enum updated; Cam can do this as part of THIS ticket (10689) (if Vysakh's ticket is approved)
- the code above will need to be updated; Cam can do this in this PR (if Vysakh's ticket is approved)
- if Vysakh's ticket is not approved for this sprint then we carry on with the current enum differences and fix it later

Reply:
Thanks you!